#! /bin/sh /usr/share/dpatch/dpatch-run
## glade.dpatch
## Gustavo Franco <stratus@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Fixes gtimelog.png path in gtimelog.glade

@DPATCH@
diff -urNad gtimelog-0.0+svn88~/gtimelog.glade gtimelog-0.0+svn88/gtimelog.glade
--- gtimelog-0.0+svn88~/gtimelog.glade	2008-01-03 23:21:37.000000000 +0100
+++ gtimelog-0.0+svn88/gtimelog.glade	2008-01-03 23:35:31.000000000 +0100
@@ -13,7 +13,7 @@
   <property name="default_height">500</property>
   <property name="resizable">True</property>
   <property name="destroy_with_parent">False</property>
-  <property name="icon">gtimelog.png</property>
+  <property name="icon">/usr/share/pixmaps/gtimelog.png</property>
   <property name="decorated">True</property>
   <property name="skip_taskbar_hint">False</property>
   <property name="skip_pager_hint">False</property>
